js/app.js

```diff
@@ -230,8 +230,9 @@ app.controller('CheckoutController', function($scope, $interval, Order, $httpPar
                 $scope.clock = $scope.order.timestamp + totalTime - Math.floor(Date.now() / 1000);
                 //Mark order as expired if we ran out of time
                 if ($scope.clock < 0) {
-                    $scope.order.status = -3;
-                    return;
+						$scope.order.status = -3;
+						return;
+						}
```

Comment:
This line has been included as an error. Using a text editor like [Sublime Text](https://www.sublimetext.com), can avoid such issues and correct the issues with spacing and indents in your code.

Reply:
ok, done.